test/shortcuts.test.js

```diff
@@ -472,6 +472,18 @@ test('[shortcuts] queue', (assert) => {
     'expected resources generated no defaults'
   );
 
+  queue = new cf.shortcuts.Queue({
+    LogicalName: 'MyQueue',
+    TopicArn: 'arn:aws:sns:us-east-1:111122223333:MyTopic'
```

Comment:
Could you add one more test case where the caller provides a reference to a topic arn, (e.g. { "Ref": "MyTopic" }) instead of providing it as a string?